### ISSUE TITLE
Ubuntu 21.10: No Release File

### ISSUE THREAD

**Issue by gregkrsak** [KrispyCamel4u/SysMonTask#96]
The installation instructions do not work on Ubuntu 21.10 (Impish).

Following these instructions: https://github.com/KrispyCamel4u/SysMonTask#installation

I get the resulting output:

**$ sudo add-apt-repository ppa:camel-neeraj/sysmontask**
```
[. . .]
Repository: 'deb http://ppa.launchpad.net/camel-neeraj/sysmontask/ubuntu/ impish main'
Description:
A window like Task Manager for Linux.
More info: https://launchpad.net/~camel-neeraj/+archive/ubuntu/sysmontask
Adding repository.
Press [ENTER] to continue or Ctrl-c to cancel.
Adding deb entry to /etc/apt/sources.list.d/camel-neeraj-ubuntu-sysmontask-impish.list
Adding disabled deb-src entry to /etc/apt/sources.list.d/camel-neeraj-ubuntu-sysmontask-impish.list
Adding key to /etc/apt/trusted.gpg.d/camel-neeraj-ubuntu-sysmontask.gpg with fingerprint 6FF464A7A524D6B532AC6ED0499E6345B743746B
[. . .]                                                             
Err:8 http://ppa.launchpad.net/camel-neeraj/sysmontask/ubuntu impish Release              
  404  Not Found [IP: 91.189.95.85 80]
[. . .]
E: The repository 'http://ppa.launchpad.net/camel-neeraj/sysmontask/ubuntu impish Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**$ sudo apt install sysmontask**
```
[. . .]
E: Unable to locate package sysmontask
```

**$ sudo apt update**
```
[. . .]
Err:8 http://ppa.launchpad.net/camel-neeraj/sysmontask/ubuntu impish Release
  404  Not Found [IP: 91.189.95.85 80]
[. . .]
E: The repository 'http://ppa.launchpad.net/camel-neeraj/sysmontask/ubuntu impish Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**$ sudo apt install sysmontask**
```
[. . .]
E: Unable to locate package sysmontask
```

**Comment by KrispyCamel4u** [comment]
Thanks.
This by this weekend it will be available.

**Comment by KrispyCamel4u** [comment]
Hi,
The build is available now.

**Comment by gregkrsak** [comment]
Confirmed fixed on Lubuntu 21.10
